src/avrdude.1

```diff
@@ -848,7 +848,12 @@ so previously entered command lines can be recalled and edited.
 .Pp
 The <addr> and <len> parameters of the read, write and erase commands can be
 negative with the same syntax as substring() computations in perl or python.
-The table below details their meaning with respect to a memory of size=0x800:
+The table below details their meaning with respect to a memory of size=0x800.
+Note that
+.Ar pos
+means positive, and
```

Comment:
Readers will be able to guess what pos or neg might mean, but the table below lacks the row with the general case when `addr` is negative, so they'd have to work that back from the couple of examples. 


| `addr` |`len` | Memory interval | Comment |
| --: | --: | --: | :-- |
| negative | positive | [size+addr, size+addr+len-1] | Start address is \|addr\| bytes below memory size |

Reply:
The problem is that I'm having a _real hard time_ getting everything to fit within 75 characters (80 - 5 indents).
Can some of the memory intervals or examples be re-written or reformatted to fir within 75 characters? I'm all ears!

```
                                             75 character limit ends here!
                                                                         |
                                                                         |
addr   len  Memory interval             Comment                          v
--------------------------------------------------------------------------
0/pos  pos  [addr, addr+len-1]          Conventional use, len=end-start+1
0/pos  neg  [addr, size+len]            End addr |len| bytes below mem size
neg    pos  [size+addr,size+addr+len-1] Start addr is |addr| bytes below mem size
neg    neg  [size+addr, size+len]       Combining above two cases
0x700  12   [0x700, 0x711]              Example
1024   -257 [0x400, 0x6ff]              Here, size of mem is 2048 or 0x800
-512   512  [0x600, 0x7ff]              Last 512 bytes
-256   -1   [0x700, 0x7ff]              Last 256 bytes
0      49   [0, 48]                     First 49 bytes
0      -1   [0, 0x7ff]                  Entire mem without knowing its size
```